### DISCUSSION TITLE
Sobre estructuras de datos a utilizar (parte 2)

### DISCUSSION THREAD

**Discussion by ManuelV03** [IIC2133-PUC/2024-1#88] [Tarea 1] [answered]
Hola, pensando en cómo hacer la parte 2 de la tarea, me surgió la duda de si es obligatorio usar árboles para todos los tipos de búsqueda, ¿o está permitido usar alguna otra estructura de datos que me permita realizar cada búsqueda con la complejidad solicitada?

**Comment by 4n1t4** [comment, marked as answer]
Holii, por motivos del avance de cronograma se espera que sea suficientemente para utilizar árboles. Ahora en caso de que por tu cuenta conozcas otra forma mas optima eres invitado a ejecutarlo. Sin embargo siempre justifica de buena manera tus elecciones dentro del informe para tenerlo como respaldo.
